SweetTweetApp/src/main/webapp/resourses/js/App.js

```diff
@@ -0,0 +1,199 @@
+        "createdAt": "2020-02-20T16:33:16",
+        "author": "Leopold",
+        "hashTags": ["#dinner", "#power"],
+        "likes": ["Lola", "Alexander T", "Dean W", "Sam W"]
+    },
+    {
+        "id": "15",
+        "description": "I just think the most difficult thing to displace is privilege.",
+        "createdAt": "2020-02-20T18:23:44",
+        "author": "Sean C",
+        "hashTags": ["#reflections"],
+        "likes": ["Lola", "Paula", "Egor B", "Dean W", "Sam W"]
+    },
+    {
+        "id": "16",
+        "description": "When I was young, even though I received so much love,\n I used to pay attention to people who disliked me. That\"s why my lyrics were so sharp and dark.",
+        "createdAt": "2020-02-20T23:48:05",
+        "author": "IU",
+        "hashTags": ["#love", "#lyrics"],
+        "likes": ["Lola Skin", "Paula", "Egor B", "Marie Curie", "Dean W", "Sam W"]
+    },
+    {
+        "id": "17",
+        "description": "The best known blues musician today is B.B. King. His fame is well-deserved.\n Born in Indianola, Mississippi in 1925, he earned the nickname \"B.B.\" (\"Blues boy\")\n while playing on radio programs in Memphis, Tennessee.",
+        "createdAt": "2020-02-21T07:32:15",
+        "author": "Alexander T",
+        "photoLink": "https://img.stereo.ru/article-covers/2018/cd94b715ef23dafa68adf62f2127b260.jpg",
+        "hashTags": ["#king", "#enjoy", "#bluise"],
+        "likes": ["Sean C", "Egor B", "Vlad K", "Sam W"]
+    },
+    {
+        "id": "18",
+        "description": "There is nothing in the world so irresistibly contagious as laughter and good humor.",
+        "createdAt": "2020-02-21T10:14:00",
+        "author": "Dickens",
+        "hashTags": ["#reflections", "#thinkaboutit"],
+        "likes": ["Lola", "Paula", "Egor B", "Dean W", "Sam W"]
+    },
+    {
+        "id": "19",
+        "description": "Kids, let\'s all get along!",
+        "createdAt": "2020-02-22T15:34:10",
+        "author": "Leopold",
+        "hashTags": ["#friendly", "#cat", "#sweety"],
+        "likes": ["Kinnoske", "Paula", "Egor B", "Dean W", "Sam W"]
+    },
+    {
+        "id": "20",
+        "description": "America’s music culture would be incomplete without\n blues music. Thought it was created in the early decades of the 20th century,\n blues music has had a huge influence on American popular music\n up to the present days. ",
+        "createdAt": "2020-02-22T20:40:00",
+        "author": "Alexander T",
+        "hashTags": ["#music", "#america", "#bluise"],
+        "likes": ["Lola Skin", "Paula", "Egor B", "Dean W", "Sam W"]
+    }
+];
+
+class App {
+
+    constructor() {
+        this.tweetList = new TweetList(POSTS.map((p) => ({ ...p, createdAt: new Date(p.createdAt ) })));
+    }
+
+    renderPosts() {
+        let view = new View();
+        view.addPostsToNewsLine(this.tweetList.getPosts(), "Alexander T");
+    }
+
+    addPost(post){
```

Comment:
Adding a single post will lead to rendering lots of posts?

Upd.: got your idea, that it clears the view before adding, but here can be a problem, that not all the posts are visible, what if pagination button was clicked before, etc.